MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -777,7 +846,29 @@
     <!-- This string is displayed in a button that when pressed allows a user to view a page that has been blocked from being displayed.-->
     <string name="performance_unblock_page">Unblock Page</string>
     <!-- This string is displayed as the title of a dialog displayed when poor web page performance has been detected. -->
-    <string name="performance_title">Poor Performance Detected</string>
+    <string name="performance_title">Poor Site Performance</string>
     <!-- This string is displayed as the message body of a dialog displayed when poor web page performance has been detected and the page has been unloaded to restore performance. -->
-    <string name="performance_message">The current web page is affecting performance and has been suspended. Reducing the window size can help improve a web page with poor performance.</string>
+    <string name="performance_message">Try reducing the window size to improve performance. Unblocking this page may crash your app.</string>
+
+    <!-- This string is displayed anywhere where we need to display the "Default" string by itself. -->
+    <string name="tag_default">Default</string>
```

Comment:
I'm not sure we want to do this.
@pike : Could you please confirm?
(also I see there's another "default" string below, so not sure this is useful)

Reply:
I've removed that string as it's not being used.